src/RestWrite.js

```diff
@@ -109,6 +115,8 @@ RestWrite.prototype.execute = function() {
       return this.deleteEmailResetTokenIfNeeded();
     })
     .then(() => {
+      delete this.data._batchCount;
```

Comment:
Is this needed?

Reply:
If I remove those then it will not pass the schema validation. When [this](https://github.com/parse-community/parse-server/blob/b611746ae5eebedc99bfa51f565615c39c146559/src/triggers.js#L281) is called via [this function](https://github.com/parse-community/parse-server/blob/b611746ae5eebedc99bfa51f565615c39c146559/src/RestWrite.js#L106), it sets this.data back to the request.body params which includes the _batchCount and _batchIndex properties. Do you have any suggestions?